tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -20,6 +20,14 @@ Verify Resource Link Http status code
         Log To Console    ${idx}. ${href} gets status code ${status}
     END
 
+Go to RHODS Dashboard, Resources And Filter By enabled state
```

Comment:
I would change the test title with something more like `Verify Use Can Filter Resources By Enabled State`. 
In addition, I think it would be good to verify other filters other than Enable state